ui/apps/routing/src/extensions/breadcrumb-navigation.tsx

```diff
@@ -172,21 +181,27 @@ const BreadcrumbNavigation: React.FC<RootExtensionProps> = props => {
     <>
       <Box direction="row" align="stretch">
         <BasicCardBox
-          onClick={() =>
-            handleNavigation(
-              matchingApp.name,
-              stack.length - 2,
-              stack[stack.length - 2].route,
-              stack[stack.length - 2].fullRoute,
-            )
+          onClick={
+            backButtonActive
+              ? () =>
+                  handleNavigation(
+                    matchingApp.name,
+                    stack[stack.length - 2].route,
+                    stack[stack.length - 2].fullRoute,
+                  )
+              : undefined
           }
           pad="medium"
           margin={{ bottom: 'xsmall', right: 'xsmall' }}
           width="auto !important"
           align="center"
           direction="row"
         >
-          <StyledBackIcon type="arrowLeft" size="sm" />
+          {backButtonActive ? (
+            <StyledBackIcon type="arrowLeft" size="sm" />
+          ) : (
+            <Icon type="akasha" size="sm" />
```

Comment:
the idea here was to have the icon of the specific app that you are on now, just like in the sidebar where each app has also an icon, not only a fixed akasha logo

Reply:
ah ok, just added it!